packages/contentstack-import/src/commands/cm/stacks/import.ts

```diff
@@ -110,18 +110,17 @@ export default class ImportCommand extends Command {
       // Note setting host to create cma client
       importConfig.host = this.cmaHost;
       contentDir = importConfig.contentDir;
-      backupDir = importConfig.backupDir;
       const managementAPIClient: ContentstackClient = await managementSDKClient(importConfig);
       const moduleImporter = new ModuleImporter(managementAPIClient, importConfig);
       await moduleImporter.start();
       log(importConfig, `The content has been imported to the stack ${importConfig.apiKey} successfully!`, 'success');
-      log(importConfig, `The log has been stored at '${path.join(importConfig.backupDir, 'logs', 'import')}'`, 'success');
+      log(importConfig, `The log has been stored at '${path.join(contentDir, 'logs', 'import')}'`, 'success');
```

Comment:
Is it content dir/ Back up dir where logs are getting created

Reply:
@shafeeqd959 content directory.